src/components/IntegrationExamples.js

```diff
@@ -0,0 +1,162 @@
+import React, { useMemo } from 'react'
+
+import { Figcaption, Microlink, MultiCodeEditor } from 'components/markdown'
+
+const DEFAULT_URL_NAME = 'microlink.io'
+const DEFAULTS = {
+  url: 'https://microlink.io'
+}
+
+const toSnakeCase = string =>
```

Comment:
we can use [lodash/snakeCase](https://lodash.com/docs/4.17.15#snakeCase) since we are using lodash in the rest of the codebase!

Reply:
Updated! We actually need `kebabCase`